.ci/lib.sh

```diff
@@ -48,15 +48,45 @@ function clone_build_and_install() {
 	popd
 }
 
+function install_yq() {
+	GOPATH=${GOPATH:-${HOME}/go}
+	mkdir -p "${GOPATH}/bin"
+	local yq_path="${GOPATH}/bin/yq"
+	[ -x  "${GOPATH}/bin/yq" ] && return
+
+	case "$(arch)" in
+	"aarch64")
+		goarch=arm64
+		;;
+
+	"x86_64")
+		goarch=amd64
+		;;
+	"*")
+		echo "Arch $(arch) not supported"
+		exit
+		;;
+	esac
+
+	local yq_url="https://github.com/mikefarah/yq/releases/download/1.14.0/yq_linux_${goarch}"
```

Comment:
You can use this API URL to download the latest version without hard-coding:

```
curl -s https://api.github.com/repos/mikefarah/yq/releases/latest|grep browser_download_url|awk '{print $2}'|tr -d '"'|grep "${goarch}"
```

Reply:
I could, but doing request to the github api may need a token at some point if is used too many times. I remember had issues in CI metrics before for that.

Reply:
@jodh-intel  I wonder if something like this may work:

```
version_url=$(curl -Ls -o /dev/null -w %{url_effective} https://github.com/mikefarah/yq/releases/latest)
version=$(basename  $version_url)

```

Reply:
Neat!

Reply:
Changed to proposed  solution in this comment.